android-vjestina-homework-movie-app-data-mock/app/src/main/java/agency/five/codebase/android/movieapp/data/database/FavoriteMovieDao.kt

```diff
@@ -0,0 +1,18 @@
+package agency.five.codebase.android.movieapp.data.database
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.Query
+import kotlinx.coroutines.flow.Flow
+
+@Dao
+interface FavoriteMovieDao {
+    @Query("SELECT * FROM DbFavoriteMovie")
+    fun favorites(): Flow<List<DbFavoriteMovie>>
+
+    @Insert
+    fun insertMovie(movie: DbFavoriteMovie)
+
+    @Query("DELETE FROM DbFavoriteMovie WHERE id = :movieId")
```

Comment:
This was actually a bad advice since now you have to pass the entire object instead of just id (I thought you could delete by primary key, sorry). You don't have to change it back, just wanted to let you know that the previous (your) implementation was better.